### ISSUE TITLE
我可以在thinkjs项目里面配置mongoDB和mysql同时使用吗

### ISSUE THREAD

**Issue by noderex** [thinkjs/thinkjs#1254]
我可以在thinkjs项目里面配置mongoDB和mysql同时使用吗？
如果可以，请问如何配置？

**Comment by lizheming** [comment]
Yes, you can. Just add both `think-mongo` and `think-model` in your `extends.js` file. More infomation can find in documentation: 
- https://thinkjs.org/zh-cn/doc/3.0/relation_model.html
- https://thinkjs.org/zh-cn/doc/3.0/mongo.html

**Comment by noderex** [comment]
谢谢，认真看了一下文档，理解了，非常感谢。
